### PR TITLE
CSP Doc Fix 

### PR DESCRIPTION
**Pull request by cnalley**
Interfaces doc ends abruptly. Copied back from the GomSpace repo converting from rst to md.

// cc @kubostech/devs

### CODE REVIEW

File: libcsp/doc/interfaces.md [hunk 1]
```diff
@@ -55,4 +55,40 @@ A new CSP buffer is preallocated with csp\_buffer\_get(). When data is received,
...
+Initialization
+--------------
+
+In order to initialize the interface, and make it available to the router, use the following function found in csp/csp_interface.h:
+
+``` sourceCode
+csp_route_add_if(&csp_if_fifo);
+```
+
+This actually happens automatically if you try to call csp_route_add() with an interface that is inknown to the router. This may however be removed in the future, in order to ensure that all interfaces are initialised before configuring the routing table. The reason is, that some products released in the future may ship with an empty routing table, which is then configured by a routing protocol rather than a static configuration.
```
Comment (CatFreed) [inline libcsp/doc/interfaces.md]: nit: spelling "unknown" "initialized".  Remove comma after "reason is,".

File: libcsp/doc/interfaces.md [hunk 2]
```diff
@@ -55,4 +55,40 @@ A new CSP buffer is preallocated with csp\_buffer\_get(). When data is received,
...
+
+In order to setup a manual static route, use the follwing example where the default route is set to the fifo interface:
```
Comment (CatFreed) [inline libcsp/doc/interfaces.md]: nit: spelling "following"

File: libcsp/doc/interfaces.md [hunk 3]
```diff
@@ -55,4 +55,40 @@ A new CSP buffer is preallocated with csp\_buffer\_get(). When data is received,
...
+
+``` sourceCode
+csp_route_set(CSP_DEFAULT_ROUTE, &csp_if_fifo, CSP_NODE_MAC);
+```
+
+All outgoing traffic except loopback, is now passed to the fifo interface's nexthop function.
```
Comment (CatFreed) [inline libcsp/doc/interfaces.md]: nit: add comma before "except"

**Comment by cnalley** [comment]
Good catch on the editing @CatFreed. My new rule "don't assume the original author edited this" :smile:.

**Pull request merged**